collector/configuration/store.json.md

```diff
@@ -42,4 +42,69 @@ The alias is used as a key reference for tags, units and types values.&#x20;
 
 Obs: For **OPC-UA** sources, the tag field should be written as the following pattern: `ns=<namespaceindex>;<type>=<value>`
 
-Instead of manually configuring this file, is also possible to use the remote control page [OPC Requests](./../remote-control/sources/opc-requests.md) to do this settings.
+### For Liverig Collector versions 5.0.0 or above
+
+Since Liverig Collector version 5.0.0, the collector can be configured to extract field from object values in **OPC-UA** sources. 
+
+**Example:**
+
+![OpcuaObjectEvent1.png](../../.gitbook/assets/OpcuaObjectEvent1.png)
+
+In this event, the OPC-UA source returned a value structured as an object with the following format:
+
+```json
+{
+  "utcTime": 133144611706210000
+}
+```
+
+To extract the field `utcTime` as the value itself we need to configure the tag using the `?field=` parameter. Example:
+`{tag}?field={path}`.
+
+So, in this example the previous tag `"ns=2;s=HelloWorld/ScalarTypes/UtcTime"` would be changed to `"ns=2;s=HelloWorld/ScalarTypes/UtcTime?field=/utcTime"`
+Resulting in the following value:
+
+![OpcuaObjectEvent2.png](../../.gitbook/assets/OpcuaObjectEvent2.png)
+
+**Example2:**
+
+![OpcuaObjectEvent3.png](../../.gitbook/assets/OpcuaObjectEvent3.png)
+
+In this event, the OPC-UA source returned a value structured as an object with the following format:
+
+```json
+{
+  "bodyType": "ByteString",
+  "encodingId": {
+    "identifier": {
+      "value": 886
+    },
+    "namespaceIndex": {
+      "value": 0
+    }
+  },
+  "decoded": {},
+  "body": {
+    "bytes": [0,0,0,0,0,0,0,0,0,0,0,0,0,0,89,64]
+  }
+}
+```
+To extract the field `endodingId/indentifier/value` as the value itself we need to configure the tag using the `?field=` parameter. Example:
+`{tag}?field={path}`.
+
+So, in this example the previous tag `"ns=2;s=HelloWorld/DataAccess/AnalogValue/0:EURange"` would be changed to `"ns=2;s=HelloWorld/DataAccess/AnalogValue/0:EURange?field=/encodingId/identifier/value"`.
+If you want to extract other fields from the same object, you can declare it as a new tag, like `"ns=2;s=HelloWorld/DataAccess/AnalogValue/0:EURange?field=/encodingId/namespaceIndex/value"`
+to extract the `encodingId/namespaceIndex/value` as a value.
+
+Resulting in the following value:
+
+![img.png](../../.gitbook/assets/OpcuaObjectEvent4.png)
+
+**Note**: Since the `tags` field from the `store.json` file is a `Map`, you need to add a new alias for each field you want to fetch. Example:
+`"RangeObject/Identifier": "ns=2;s=HelloWorld/DataAccess/AnalogValue/0:EURange?field=/encodingId/identifier/value"` and
+`"RangeObject/namespaceIndex": "ns=2;s=HelloWorld/DataAccess/AnalogValue/0:EURange?field=/encodingId/namespaceIndex/value"`
+
+
+{% hint style="info" %} 
+Instead of manually configuring this file, is also possible to use the remote control page [OPC Requests](./../remote-control/sources/opc-requests.md) to easily change settings.
```

Comment:
```suggestion
Instead of manually configuring this file, is also possible to use the remote control page [OPC Requests](./../remote-control/sources/opc-requests.md) to change the settings easily.
```